CHANGELOG.md

```diff
@@ -153,60 +106,100 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ### Fixed
-- [MiqQueue] Format .format_full_log_msg [(#19296)](https://github.com/ManageIQ/manageiq/pull/19296)
-- Set default playbook service timeout to 100 minutes [(#19279)](https://github.com/ManageIQ/manageiq/pull/19279)
-- Remove value2tag [(#19275)](https://github.com/ManageIQ/manageiq/pull/19275)
-- Make description unique for tenant groups in MiqGroup [(#19272)](https://github.com/ManageIQ/manageiq/pull/19272)
-- Schedule compliance purging [(#19264)](https://github.com/ManageIQ/manageiq/pull/19264)
-- Do not calculate backlog when subscription not active [(#19254)](https://github.com/ManageIQ/manageiq/pull/19254)
-- Fixed 'TypeError'  in ApplicationCable::Connection#connect [(#19248)](https://github.com/ManageIQ/manageiq/pull/19248)
-- Fix Embedded Ansible Plays in the Job UI (v2/rebased with master) [(#19247)](https://github.com/ManageIQ/manageiq/pull/19247)
-- Workflow cloud_tenant fix [(#19237)](https://github.com/ManageIQ/manageiq/pull/19237)
-- Do not  attempt to calculate backlog unless subscription status is "replicating" [(#19234)](https://github.com/ManageIQ/manageiq/pull/19234)
-- Update the button order on copy [(#19227)](https://github.com/ManageIQ/manageiq/pull/19227)
+* Update translations for ivanchuk [(#19301)](https://github.com/ManageIQ/manageiq/pull/19301)
+* Service retirement request should be per region. [(#19143)](https://github.com/ManageIQ/manageiq/pull/19143)
+* Fixed MiqSearch.seed  when search name was changed and corrected typo miq_search.yml [(#19151)](https://github.com/ManageIQ/manageiq/pull/19151)
+* Schedule compliance purging [(#19264)](https://github.com/ManageIQ/manageiq/pull/19264)
+* Remove unnecessary log message. [(#19299)](https://github.com/ManageIQ/manageiq/pull/19299)
+* Add check for nil group to system_context_requester [(#19309)](https://github.com/ManageIQ/manageiq/pull/19309)
+* Drop load balancer from the custom buttons tree in automate [(#19331)](https://github.com/ManageIQ/manageiq/pull/19331)
+* [EmbeddedAnsible] Force embedded_ansible role for workflow [(#19187)](https://github.com/ManageIQ/manageiq/pull/19187)
+* Use find_by to skip exceptions in Export of schedules [(#19191)](https://github.com/ManageIQ/manageiq/pull/19191)
+* Fix inverse_of for DistributedVirtualSwitch [(#19221)](https://github.com/ManageIQ/manageiq/pull/19221)
+* [V2V] Allow active InfraConversionJob to be throttled [(#19277)](https://github.com/ManageIQ/manageiq/pull/19277)
+* Update translations for ivanchuk [(#19301)](https://github.com/ManageIQ/manageiq/pull/19301)
+* [EmbeddedAnsible] Fix job_plays for API [(#19320)](https://github.com/ManageIQ/manageiq/pull/19320)
+* Fix an exception calling compliance_purge_timer [(#19326)](https://github.com/ManageIQ/manageiq/pull/19326)
+* [V2V]Fix create_job in JobProxyDispatcher specs [(#19461)](https://github.com/ManageIQ/manageiq/pull/19461)
+
+### Removed
+Remove deprecated VMWare Hosts [(#19159)](https://github.com/ManageIQ/manageiq/pull/19159)
```

Comment:
Missing a `*` here

Reply:
Correct!  The changelog did not have all of the items for anything after Ivanchuk-1.